packages/@aws-cdk/aws-kms/test/key.test.ts

```diff
@@ -245,6 +245,106 @@ describe('key policies', () => {
     });
   });
 
```

Comment:
Also run the test cases (locally) reported in the original issue in the s3 and codepipeline module with the feature flag set and confirm that this is fixed.

Reply:
Yup, already done prior to posting. :)